cookbook/agent_os/client/01_basic_client.py

```diff
@@ -0,0 +1,52 @@
+"""
+Basic AgentOSClient Example
+
+This example demonstrates how to use AgentOSClient to connect to
+a remote AgentOS instance and perform basic operations.
+
+Prerequisites:
+1. Start an AgentOS server:
```

Comment:
Maybe make a helper file in this directory to spin up an AgentOS that would work for all the cookbooks